Blish HUD/GameServices/ArcDps/V2/Models/Unofficial Extras/NpcMessageInfo.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Blish_HUD.GameServices.ArcDps.V2.Models.UnofficialExtras {
+    public struct NpcMessageInfo {
+        public string CharacterName { get; set; }
+
+        public string Message { get; set; }
+        
+        public ulong TimeStamp { get; set; }
```

Comment:
```suggestion
        /// <summary>
        /// Time since epoch in nanoseconds.
        /// This can be used to sort messages, when they are out of order.
        /// </summary>
        public ulong TimeStamp { get; set; }
```
I feel like it's helpful to document the `TimeStamp`, because the different time formats used all over ArcDPS and arcdps_unofficial_extras can quickly become confusing.